frontend/www/js/omegaup/components/DatePicker.vue

```diff
@@ -27,6 +28,8 @@ export default class DatePicker extends Vue {
   @Prop({ default: true }) enabled!: boolean;
   @Prop({ default: T.datePickerFormat }) format!: string;
   @Prop({ default: false }) isInvalid!: boolean;
+  @Prop({ default: new Date(8640000000000000).toISOString().split('T')[0] })
+  max!: Date;
```

Comment:
```suggestion
  @Prop({ default: null }) max!: Date | null;
```

it's probably better to disable the maximum by explicitly using `null` instead of a magic constant if nothing was passed.

Reply:
In fact, I used `new Date()` at first, but it doesn't work. It turns out that the <input type="date"> element only accepts dates in **ISO format**. That's why I intentionally convert it to ISOString.
A blog on [stackoverflow.com](https://stackoverflow.com/questions/38638424/html5-input-type-date-disable-dates-before-today) also explains this.

Reply:
ok. I changed the data type of _max_ to string, and use max='' to disable maximum.

Reply:
can this please be kept as `Date | null`? I left another comment with a bit more explanation.